src/screens/UserPortal/Volunteer/Groups/Groups.spec.tsx

```diff
@@ -15,6 +21,8 @@ import Groups from './Groups';
 import type { ApolloLink } from '@apollo/client';
 import { MOCKS, EMPTY_MOCKS, ERROR_MOCKS } from './Groups.mocks';
 import useLocalStorage from 'utils/useLocalstorage';
+import { vi } from 'vitest';
+import '@testing-library/jest-dom';
```

Comment:
Why was this added if we are migrating from jest to vitest?

Reply:
Yes,  this line is unnecessary  . 
I have added it because earlier code without this line was giving error.
but now
`setupFiles: 'vitest.setup.ts',`
line in vitest.config.ts  does the same job